packages/cfd/src/features/types/components.types.ts

```diff
@@ -0,0 +1,352 @@
+    type: 'displayText' | 'displayIcons';
+};
+
+export type TJurisdictionCardSection = {
+    key: string;
+    title: string;
+    description?: string;
+    clickable_description?: Array<TClickableDescription>;
+    title_indicators?: TJurisdictionCardSectionTitleIndicators;
+};
+
+export type TJurisdictionCardVerificationStatus = 'Pending' | 'Verified' | 'Failed' | 'Default';
+
+export type TJurisdictionCardItemVerificationItem =
+    | 'selfie'
+    | 'document_number'
+    | 'identity_document'
+    | 'name_and_address'
+    | 'not_applicable';
+
+export type TJurisdictionCardItemVerification = Array<TJurisdictionCardItemVerificationItem>;
+
+export type TJurisdictionCardItems = {
+    header: string;
+    over_header?: string;
+    is_over_header_available: boolean;
+    synthetic_contents: TJurisdictionCardSection[];
+    financial_contents: TJurisdictionCardSection[];
+    swapfree_contents?: TJurisdictionCardSection[];
+    synthetic_verification_docs?: TJurisdictionCardItemVerification;
+    financial_verification_docs?: TJurisdictionCardItemVerification;
+};
+
+export type TJurisdictionCardParams = {
+    toggleDynamicLeverage: React.MouseEventHandler<HTMLSpanElement>;
+};
+
+export type TJurisdictionVerificationSection = {
+    icon: string;
+    text: string;
+};
+
+export type TJurisdictionVerificationItems = {
+    selfie?: TJurisdictionVerificationSection;
+    not_applicable?: TJurisdictionVerificationSection;
+    document_number?: TJurisdictionVerificationSection;
+    name_and_address?: TJurisdictionVerificationSection;
+    identity_document?: TJurisdictionVerificationSection;
+};
+
+type TJurisdictionVerificationColors = 'yellow' | 'red' | 'green';
+
+export type TJurisdictionVerificationStatus = {
+    icon: string;
+    text: string;
+    color: TJurisdictionVerificationColors;
+};
+
+export type TExistingData = DetailsOfEachMT5Loginid & DetailsOfEachMT5Loginid[];
+
+export type TCFDAccountCard = {
+    type: TType;
+    title: string;
+    platform: TCFDPlatform;
+    descriptor: string;
+    is_virtual?: boolean;
+    is_hovered?: boolean;
+    has_banner?: boolean;
+    is_disabled: boolean;
+    is_logged_in: boolean;
+    commission_message: string;
+    has_real_account?: boolean;
+    is_button_primary?: boolean;
+    onSelectAccount: () => void;
+    has_cfd_account_error?: boolean;
+    is_accounts_switcher_on?: boolean;
+    button_label?: string | JSX.Element;
+    existing_accounts_data?: TExistingData | null;
+    toggleAccountsDialog?: (arg?: boolean) => void;
+    onClickFund: (arg: DetailsOfEachMT5Loginid) => void;
+    toggleShouldShowRealAccountsList?: (arg: boolean) => void;
+    specs?: { [key: string]: { key: () => string; value: () => string } };
+    onPasswordManager: (
+        arg1: string | undefined,
+        arg2: string,
+        group: TCFDPasswordReset['account_group'],
+        arg4: string,
+        arg5: string | undefined
+    ) => void;
+};
+
+export type TTradingPlatformAccounts = {
```

Comment:
We already have this in the api-types why here again ? 🤔

Reply:
I have not found TradingPlatformAccounts type anywhere in the project.

Reply:
This type looks like api-types, if it's not from api, please remove the comments to keep the consistency of through the file